### DISCUSSION TITLE
The editor UI's opacity on hover makes it look like it's 'flashing'

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#28581] [Ideas] [answered]
The editor UI, particularly in areas far away from the cursor, looks like it's 'flashing'. This is primarily noticable on a large monitor, where the editor UI is mainly in your peripheral vision. If you watch the video below without going fullscreen, it probably looks like a non-issue, so I do suggest opening the editor and trying it yourself, or at least viewing the video in fullscreen on a large monitor. A bright background with a low background dim helps to accentuate the issue

https://github.com/ppy/osu/assets/49025619/a91303ba-c613-422d-b4da-b15b3073ca4f

I think that having the UI only be opaque surrounding the cursor would minimise this effect. I don't know how technically feasible this is, but I do think that it looks a lot better

![editor_opacity_example](https://github.com/ppy/osu/assets/49025619/306d5c4f-9afe-4abf-a998-5fd5e93e113e)

**Comment by peppy** [comment]
I'm not sure I agree this is an issue.

**Reply by tovc** [reply]
That's why I created an 'idea' instead of an 'issue'. I personally find this to be very distracting - more so than just having the UI be opaque or transparent all the time

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/28787

**Comment by RatinFX** [comment]
Instead of having another effect on top, I would rather have the option to:
- manually toggle the transparency,
- and also turn on/off the "opacity change on hover" effect.

I agree it can become a bit flashy when you generally move your cursor around.

For example, moving between an open context menu (right-click), the selected object's buttons, the corner rotate button, the scaling anchor points, and any of the side panels.

> https://github.com/ppy/osu/assets/38523955/4fd19aa9-bc09-4a14-85de-b3fa48abf8bc
